contract-creator/src/org/jetbrains/ruby/runtime/signature/server/SignatureServer.kt

```diff
@@ -70,36 +84,44 @@ object SignatureServer {
 
```

Comment:
|`line 76`|
|---|
I suggest addressing that in another PR in order to increase the possiblity of that one be merged :)